### PR TITLE
tox.ini_issues#30

### PR DESCRIPTION
**Pull request by antotaverna**
Copy from project Grispy.

### CODE REVIEW

**Comment by antotaverna** [comment]
I don't know if I must put the coverage ([testenv:coverage]) in the script tox.ini, or is enough for now.

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+[tox]
+envlist = py38, style, coverage, docs, py37, py36, docstyle
+
+
+[testenv]
+deps =
+    ipdb
+    pytest
+    scipy
+    textdistance
```
Comment (BrunoSanchez) [inline tox.ini:11]: This is not needed

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+description = "Invoke sphinx-build to build th HTML docs"
+whitelist_externals =
+    make
+usedevelop = True
+skip_install = False
+changedir = docs
+deps =
+    -r {toxinidir}/docs/requirements.txt
+commands =
+    sphinx-build -W -b html -d {envtmpdir}/doctrees source {envtmpdir}/html'''
```
Comment (BrunoSanchez) [inline tox.ini:53]: Is this to make it commented?

**Review by BrunoSanchez — CHANGES_REQUESTED** [review]

File: tox.ini (hunk 1 above)
Comment (BrunoSanchez) [reply tox.ini:11]: Sorry I meant the `textdistance` package

**Comment by BrunoSanchez** [comment]
Yes, activate coverage please

File: tox.ini (hunk 2 above)
Comment (antotaverna) [reply tox.ini]: Yes, but I thought I could do it the same way as Python. I erased it.

**Review by BrunoSanchez — APPROVED** [review]
This looks better. We always can add more stuff in here.
Keep it simple right now.

I am accepting this merge.

**Pull request merged**